UKHomeOffice/ide-controller
# COP 1977: Adjust layout to centre (1)

## Description
Move main content to the centre of the display. 

## To test
- Pull and run dev `npm ci && npm run dev`
- If the controller displays blank, refrest the page `Cmd + r` or `Ctr + r`
- The main content of should be shifted to the centre of the screen. 

## Developer Checklist

\* Required
